### PR TITLE
Improve Uses in several areas

### PR DESCRIPTION
**Pull request by Grygon**
- [x] Some UI Improvements
- [x] Automatic Uses tracking
- [x] Add setting for Uses
- [x] Counters Rework

### CODE REVIEW

**Comment by BoltsJ** [comment]
Also, It would be nice to have a way to remove counters. At least for manually added ones

**Comment by Grygon** [comment]
Looks like a commit is missing, some of this was addressed... but good callout on the FormApplication, just copied the Bonus Editor's stuff and that looks better.

File: public/templates/window/counter.hbs [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+    </div>
+    <div class="dialog-buttons flexrow">
+        {{#each buttons as |button id|}}
+        <button class="dialog-button" data-button="{{id}}">
+            {{{button.icon}}}
+            {{{button.label}}}
+        </button>
+        {{/each}}
+    </div>
+</form>
```
[the remark this answers is not in the document]
Comment (Grygon) [reply public/templates/window/counter.hbs:16]: Oh, it was the issue with the .hbs that Foundry uses not actually being in the repository...

File: public/templates/window/automation-config.hbs [hunk 2]
```diff
@@ -49,6 +49,17 @@
...
+    <label>{{ localize "lancer.automation.limited_loading" }}</label>
+    <input
+        type="checkbox"
+        name="limited_loading"
+        {{#if limited_loading}}checked{{/if}}
+        {{#unless enabled}}disabled{{/unless}}
+        data-dtype="Boolean"
+    />
+    <p class="notes">{{ localize "lancer.automation.limited_loading-desc" }}</p>
+  </div>
```
[the remark this answers is not in the document]
Comment (Grygon) [reply public/templates/window/automation-config.hbs:62]: We're only about 1/2 the height of the Core Settings, might be good to try and think about that but got a decent ways to go before we're the ones causing trouble.

File: src/module/actor/lancer-actor-sheet.ts [hunk 3]
```diff
@@ -223,7 +227,33 @@ export class LancerActorSheet<T extends LancerActorType> extends ActorSheet<
...
+    elements.on("click", async ev => {
+      ev.stopPropagation();
+
+      const params = ev.currentTarget.dataset;
+      const data = await this.getDataLazy();
+      if (params.path && params.writeback) {
+        const item = resolve_dotpath(data, params.path) as Counter;
+        const writeback = resolve_dotpath(data, params.writeback) as RegEntry<any>
+        const available = params.available === "true";
+
```
[the remark this answers is not in the document]
Comment (BoltsJ) [reply src/module/actor/lancer-actor-sheet.ts:241]: There's a `.prettierrc.json` in the root.

[the remark this answers is not in the document]
Comment (Grygon) [reply src/module/actor/lancer-actor-sheet.ts:241]: ...I'm gonna go die in a hole now

**Pull request merged**